test_runner/regress/test_gc_cutoff.py

```diff
@@ -29,15 +24,18 @@ def test_gc_cutoff(neon_env_builder: NeonEnvBuilder, pg_bin: PgBin):
             "image_creation_threshold": "2",
         }
     )
+
+    pageserver_http = env.pageserver.http_client()
+    pageserver_http.configure_failpoints(("after-timeline-gc-removed-layers", "exit"))
```

Comment:
This is now enabled before the `pgbench -i -s10`.

Reply:
47b3f8461

Reply:
[47b3f84](https://github.com/neondatabase/neon/pull/5033/commits/47b3f8461b5aa81fc5ad17fbf9f4fc2241b253ee)